### PR TITLE
Fix empty JSON support

### PR DESCRIPTION
**Pull request by hyperized**
Fix empty json, rename phpunit.xml to phpunit.xml.dist and add tests to confirm empty json works

### CODE REVIEW

**Comment by hyperized** [comment]
Verified this working with:

```sh
hello-world-as-a-service on  codeQualityReport [?] via 🐘 v7.4.2 at ☸️  isset-cloud
➜ docker run --rm -t -v $(pwd):/project -w /project converter convert --phpstan --phpstan-json-file=phpstan.json --psalm --psalm-json-file=psalm.json
Converting PHPStan via phpstan.json
Converting Psalm via psalm.json
Writing output to code-climate.json

hello-world-as-a-service on  codeQualityReport [?] via 🐘 v7.4.2 at ☸️  isset-cloud
➜ cat code-climate.json
[
    {
        "description": "(Psalm) Cannot concatenate with a possibly null null|string",
        "fingerprint": "e27527712d58b2406ee486e2b7a40f95",
        "location": {
            "path": "src/HelloWorld.php",
            "lines": {
                "begin": 18,
                "end": 18
            }
        }
    }
]%
```

File: tests/PHPStan/PHPStanConverterTest.php [hunk 1]
```diff
@@ -8,6 +8,23 @@
...
+        $jsonInput = file_get_contents(__DIR__ . '/fixtures/empty.json');
+        $jsonDecodedInput = json_decode($jsonInput);
+
+        // When
+        $validator = new PHPStanJsonValidator($jsonDecodedInput);
+        $converter = new PHPStanConvertToSubset($validator, $jsonDecodedInput);
+        $converter->convertToSubset();
+
+        // Then
+        var_dump($converter);
```
Comment (hyperized) [inline tests/PHPStan/PHPStanConverterTest.php]: whoops

**Comment by ttomdewit** [comment]
Thank you very much!
